protocols/randhound/utils.go

```diff
@@ -90,17 +90,8 @@ func (rh *RandHound) hash(bytes ...[]byte) []byte {
 	return abstract.Sum(rh.Node.Suite(), bytes...)
 }
 
-func (rh *RandHound) nodeIdx() uint32 {
-	return uint32(rh.Node.TreeNode().EntityIdx)
-}
-
-func (rh *RandHound) sendToChildren(msg interface{}) error {
-	for _, c := range rh.Children() {
-		if err := rh.SendTo(c, msg); err != nil {
-			return err
-		}
-	}
-	return nil
+func (rh *RandHound) index() uint32 {
+	return uint32(rh.Node.Index())
```

Comment:
Mostly because it's annoying to Marshal / Hash `int`. So I decided to use `uint32` everywhere in RandHound for indices.